app/controllers/users_controller.rb

```diff
@@ -30,20 +30,28 @@ def show
   end
 
   def update
-    user = User.find(user_params[:email])
-    if user.update(user_params)
-      render json: user, status: :ok
-    else
-      render json: user.errors, status: :unprocessable_entity
+    user = User.find_by(email: user_params[:email])
```

Comment:
Should we include this in the rescue block to cover any possible exception inside the `find` method? The same for `delete`.